.github/workflows/build.yml

```diff
@@ -14,13 +34,24 @@ jobs:
       contents: read
       packages: write
     steps:
-      - uses: actions/checkout@v4
-      - uses: docker/metadata-action@v3
-        id: meta
+      # By setting an env variable using the shell we can use the value of an
+      # existing env variable in the new variable's value.
+      - run: "echo bigquery_emulator_version=${{ format( '{0}-{1}', 
```

Comment:
If you introduce a variable, its name should be consistent with other variables that we introduced earlier.